import_data/tasks/tasks.py

```diff
@@ -756,7 +759,14 @@ def init_osm_update(ctx):
     latest state.txt file before .pbf timestamp
     """
     logging.info("initializing osm update from osm file timestamp:")
-    ctx.run(f"mkdir -p {ctx.update_tiles_dir}")
+
+    # In containers with restricted permissions, this 'mkdir' command may fail even
+    # though the folder already exists. That's why the error is ignored.
+    # In most cases, this folder is the mount point of the volume "update_tiles_data",
+    # but creating 'update_tiles_dir' may be necessary in a different volume configuration.
+    # If this directory cannot be created, thistask will fail anyway on 'write_new_state()'.
+    ctx.run(f"mkdir -p {ctx.update_tiles_dir} || true")
```

Comment:
Is this something that could become fragile with time / in another context? If this is the case we could either add a warning or check if the repository already exists. If it is not the case I understand that it may be a bit painful for only few benefits :wink: